sources/common/toddl.go

```diff
@@ -199,3 +212,39 @@ func cvtIndexes(conv *internal.Conv, spTableName string, srcTable string, srcInd
 	}
 	return spIndexes
 }
+
+func SrcTableToSpannerDDL(conv *internal.Conv, toddl ToDdl, srcTable schema.Table) error {
+	err := SchemaToSpannerDDLHelper(conv, toddl, srcTable)
+	if err != nil {
+		return err
+	}
+	for srcTableName, srcTable2 := range conv.SrcSchema {
```

Comment:
Rename to something meaningful rather than srcTable2

Reply:
Updated to sourceTable